.travis.yml

```diff
@@ -1,54 +1,54 @@
-sudo: required
-dist: xenial
-
-language: java
-
-jdk: oraclejdk8
-
-cache:
-  directories:
-    - $HOME/.m2
- 
-env: DISPLAY=:99.0
-
-install: true
-
-addons:
-  apt:
-    packages:
-      - metacity
-
+language: generic
+env:
+  global:
+    - ORG="devonfw"
+    - EMAIL="icsddevonfwsupport.apps2@capgemini.com"
+    - REPO_SOURCE="tools-cobigen"
+    - REPO_DEST="tools-cobigen.wiki"
+    - REPO_CONSOLIDATE="devonfw-guide"
+    - GH_REPO_SOURCE="github.com/${ORG}/${REPO_SOURCE}.git"
+    - GH_REPO_DEST="github.com/${ORG}/${REPO_DEST}.git"
+    - GH_REPO_CONSOLIDATE="github.com/${ORG}/${REPO_CONSOLIDATE}.git"
+    - secure: SyWbv1nirguuzC0oUMBh2IlBbGd2DpxXHBGhbANm+OozF2iXAfLtw1IKTiUxIK/X0abx2PeWczCtouTlg4HLLREsAgJ1kZRVRDdoDJxal3UCGREo6eCwHnRMsXPBSkifcoMuipD9tbhs122aMlHpDO60NAtZtUZbVtnrlB9etcM=
+stages:
+  - name: Documentation
+    if: (branch = develop) AND (type = push)
```

Comment:
In CobiGen we don't have a `develop` branch. I think here you should use `master`.

Reply:
Branch updated!